CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog for NXP i.MX RT1060 Featured FreeRTOS IoT Integration
 
+## v202407.00 ( July 2024 )
+
+- [#48](https://github.com/FreeRTOS/iot-reference-nxp-rt1060/pull/48) Update third party submodule
+  * [Unity v2.6.0](https://github.com/ThrowTheSwitch/Unity/tree/v2.6.0)
+  * [mbedtls v2.28.8](https://github.com/Mbed-TLS/mbedtls/tree/v2.28.8)
+- [#47](https://github.com/FreeRTOS/iot-reference-nxp-rt1060/pull/47) Update integration test to 202406.00 version
+- [#45](https://github.com/FreeRTOS/iot-reference-nxp-rt1060/pull/45) Switch to modular OTA and aws-iot-core-mqtt-file-streams-embedded-c. Update Long Term Support (LTS) libraries to 202406.00.

```

Comment:
nit - Update Long Term Support (LTS) libraries to 202406.00 - LTS:

Reply:
Update in new commit.